apps/dispatch/src/app/app.component.html

```diff
@@ -34,21 +34,15 @@
 
     <biosimulations-hero-banner-util-button></biosimulations-hero-banner-util-button>
 
-    <biosimulations-hover-open-menu [href]="config.docsUrl" [disabled]="true">
-      <biosimulations-topbar-menu-item heading="Learn" icon="idea" id="trigger"> </biosimulations-topbar-menu-item>
-    </biosimulations-hover-open-menu>
-
     <biosimulations-hover-open-menu
-      [href]="appRoutes.simulatorsApp"
       [disabled]="true"
-      matTooltip="Browse simulator tools">
-      <biosimulations-topbar-menu-item heading="BioSimulators" icon="link" id="trigger">
-      </biosimulations-topbar-menu-item>
+      [href]="appRoutes.simulatorsApp"
+      matTooltip="Browse simulation projects">
```

Comment:
think it should be **"Browse simulator tools"** if the link is `appRoutes.simulatorsApp`

Reply:
Yes!! Thanks!